apps/monitoring/perftest/01/dynatrace-operator.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dynatrace-operator
+  namespace: monitoring
```

Comment:
Is this being tested in monitoring namespace first before changing to dynatrace?

Reply:
Ideally like to test under the current monitoring namespace. I did try to use dynatrace but this kept giving me an error.

Reply:
I attempted to change namespace and this is where the error occurs.